src/dotnet/MulticoreJitActivator.cs

```diff
@@ -0,0 +1,46 @@
+// Copyright (c) .NET Foundation and contributors. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System;
+using System.IO;    
+using System.Runtime.Loader;
+using Microsoft.DotNet.Cli.Utils;
+using Microsoft.DotNet.Tools.Common;
+using Microsoft.Extensions.PlatformAbstractions;
+using NuGet.Frameworks;
+
+namespace Microsoft.DotNet.Cli
+{
+    public class MulticoreJitActivator
+    {
+        public bool TryActivateMulticoreJit()
+        {
+            var disableMulticoreJit = IsMulticoreJitDisabled();
```

Comment:
What is the recommendation around ```var``` for this repo? In CoreFx, if the type of var cannot be immediately discerned then the type needs to be specified; is that the same here?